contracts/modules/STO/STO.sol

```diff
@@ -45,20 +31,12 @@ contract STO is ISTO, STOStorage, Module, Pausable {
 
     /**
      * @notice Pause (overridden function)
+     * @dev Only securityToken owner restriction applied on the super function
      */
     function pause() public {
```

Comment:
`module.sol` already has this function, why do we need it here?

Reply:
This is because we have a special condition in pause function for the STO.